Dockerfile.mysql

```diff
@@ -0,0 +1,10 @@
+FROM mysql:latest
+
+ENV MYSQL_PASSWORD=1234
```

Comment:
The `MYSQL_PASSWORD` environment variable is not used by the official MySQL Docker image. You should remove this line unless you have a specific reason to include it.